libqtile/utils.py

```diff
@@ -239,16 +234,43 @@ def send_notification(title, message, urgent=False, timeout=10000, id=None):
     passed when calling this function again to replace that notification. See:
     https://developer.gnome.org/notification-spec/
     """
-    if _can_notify and Notify.get_server_info()[0]:
-        notifier = Notify.Notification.new(title, message)
-        if urgent:
-            notifier.set_urgency(Notify.Urgency.CRITICAL)
-        notifier.set_timeout(timeout)
-        if id is None:
-            id = randint(10, 1000)
-        notifier.set_property('id', id)
-        notifier.show()
-        return id
+    id = randint(10, 1000) if id is None else id
+    urgency = 2 if urgent else 1
+
+    try:
+        asyncio.create_task(_notify(title, message, urgency, timeout, id))
```

Comment:
You might want to `asyncio.get_running_loop()` here instead so if `create_task` throws an unrelated `RuntimeError` it isn't swallowed.

Reply:
Interesting. `asyncio.get_running_loop()` also throw `RuntimeError` if there's no eventloop but  I guess your point is that it's calling an internal `asyncio` method so it only throws that error if there's no eventloop. 

Makes sense. Will tweak the code.